app/controllers/orgs/rosters_controller.rb

```diff
@@ -398,6 +401,12 @@ def user_google_classroom_credentials
     def redirect_if_roster_exists
       redirect_to roster_url(current_organization) if current_organization.roster.present?
     end
+
+    def google_classroom_ensure_no_roster
+      return unless current_organization.roster
+      redirect_to edit_organization_path(current_organization),
+        alert: "Google Classroom connection failed as you already have a roster. Please delete roster and try again."
```

Comment:
This is _very_ pedantic, but maybe we should present this alert in a friendlier manner. Maybe something like


> "We are unable to link your organization to Google Classroom because a roster already exists for this classroom organization. Please delete your current roster and try again."

Reply:
Yeah I had a linter issue with how long that message was so I decided to shorten it. I can change it back.